main/src/cgeo/geocaching/CacheListActivity.java

```diff
@@ -521,7 +521,12 @@ public void onCreate(final Bundle savedInstanceState) {
             initActionBarSpinner();
         }
 
-        currentLoader = (AbstractSearchLoader) getSupportLoaderManager().initLoader(type.getLoaderId(), extras, this);
+        try {
+            currentLoader = (AbstractSearchLoader) getSupportLoaderManager().initLoader(type.getLoaderId(), extras, this);
+        } catch (final NullPointerException e) {
```

Comment:
I don't think it is an awfully good idea to blindly swallow an exception here. If the loader is 'not needed', we should either detect it and avoid calling the creation.

Reply:
The loader is not needed, but the initialization.
Now I found the (hopefully) a correct implementation.